py_gtfs_rt_ingestion/dev_test_setup.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+import sys
+import logging
+from typing import NamedTuple
+import os
+import boto3
+import random
+
+from py_gtfs_rt_ingestion import file_list_from_s3
+
+GTFS_BUCKET = 'mbta-gtfs-s3'
+DEV_INGEST_BUCKET = 'mbta-ctd-dataplatform-dev-incoming'
+DEV_EXPORT_BUCKET = 'mbta-ctd-dataplatform-dev-springboard'
+DEV_ARCHIVE_BUCKET = 'mbta-ctd-dataplatform-dev-archive'
+DEV_ERROR_BUCKET = 'mbta-ctd-dataplatform-dev-error'
+
+logging.basicConfig(level=logging.WARNING)
+
+class SetupArgs(NamedTuple):
+    dev_prefix: str
+    src_prefix: str
+    objs_to_copy: int
+
+def parseArgs(args) -> dict:
+    """
+    parse input args from the command line and generate an event dict in the
+    format the lambda handler is expecting
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--dev-prefix',
+        dest='dev_prefix',
+        type=str,
+        default='lamp/',
```

Comment:
i think you can drop the trailing slash here.

Reply:
It works better for the script to leave it on, but I moved this into a constant variable instead of a command line argument, since it should never really change.